source/segment_context.cc

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 Rei Shimizu
```

Comment:
Just as a reminder, if this is set as copyright by individual, please make sure the following CI would check this.

I don't want this project ends by copyright from various people.

Also, it is common, copyright of the SkyAPM virtual org. It is up to you.

Reply:
Changed from my name to `SkyAPM`. right?

Reply:
If that makes sense to you, my point is, we need a consistent header for all contributors in the future, and maybe we want to donate it to the ASF later.

Reply:
Sure. Thank you for pointing out. For now, we should suit them as `SkyAPM`. When we'd like to donate this to ASF, we will change to `Apache Software Foundation (ASF)`.

Reply:
Yes. That is the thing after the donation.